stl/base.py

```diff
@@ -413,16 +412,16 @@ def __str__(self):
 
   def __getitem__(self, key):
     if key not in self.fields:
-      raise AttributeError("No field exists in role '%s': %s" %
-                           (self.name, key))
+      raise AttributeError("No field exists in role '%s': %s" % (self.name,
```

Comment:
I did. I'm okay with the slightly weird formatting; it's better than correcting gpylint every time we want to run it I think.